src/CMakeLists.txt

```diff
@@ -403,6 +403,73 @@ if(WITH_DPDK)
   list(APPEND ceph_common_deps common_async_dpdk)
 endif()
 
+if(WITH_PMDK)
+  # used in src/librbd/CMakeLists.txt and src/os/CMakeLists.txt
```

Comment:
i think this comment is not necessary, if user/developer wants to understand where `WITH_PMDK` is used, he/she can aways grep the source tree.

Reply:
Done